pipeline_tools/input_utils.py

```diff
@@ -21,6 +21,27 @@ def get_sample_id(bundle):
     return sample_id
 
 
+def get_expected_cell_count(bundle):
+    """Return the total estimated cells from the given bundle, otherwise use the same default value as CellRanger
+    (3000 cells): https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count
```

Comment:
nit: link to a pinned version of cellranger here: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/3.0/using/count